agent_plugin_builder/vendor_dirs.py

```diff
@@ -110,6 +100,16 @@ def _build_bash_command(command: str) -> str:
     return f"/bin/bash -c {quote(command)}"
 
 
+def _run_container_with_plugin_dir(image: str, command: str, plugin_dir: Path):
```

Comment:
Missing return type

Reply:
The return type is `Container`

Reply:
I think not https://docker-py.readthedocs.io/en/stable/containers.html

Reply:
Hmm, you are right. The names of variables we're using to capture the result of this call are misleading. I think we should either:
1. Change the variable names, or
2. Output a container, and use the `wait()` and `logs()` methods to do what we need